zerver/lib/push_notifications.py

```diff
@@ -566,6 +567,8 @@ def get_apns_alert_subtitle(message: Message) -> str:
     """
     if message.trigger == "mentioned":
         return message.sender.full_name + " mentioned you:"
+    elif message.trigger == "wildcard_mentioned":
+        return message.sender.full_name + " mentioned everyone:"
```

Comment:
@gnprice @rishig FYI: This is the text that will appear for wildcard mentions (`@**all**`) in mobile push notifications.  Thoughts on whether this should say "mentioned you" or "mentioned everyone"?

While we're touching this -- I assume I should get these subtitles tagged for translation?

Reply:
I like "mentioned everyone"